src/Symfony/Bundle/FrameworkBundle/EventListener/SuggestMissingPackageSubscriber.php

```diff
@@ -33,7 +33,7 @@ final class SuggestMissingPackageSubscriber implements EventSubscriberInterface
             '_default' => ['Doctrine ORM', 'symfony/orm-pack'],
         ],
         'generate' => [
```

Comment:
shouldnt we just remove this entry?

Reply:
Maybe yes, might still be helpful when upgrading though